api/src/main/java/io/strimzi/api/kafka/model/kafka/KafkaClusterSpec.java

```diff
@@ -161,9 +161,9 @@ public void setLogging(Logging logging) {
         this.logging = logging;
     }
 
-    @Description("The number of pods in the cluster.")
+    @Description("The number of pods in the cluster. " +
+            "This property is required when Node Pools are not used.")
```

Comment:
```suggestion
            "This property is required when node pools are not used to set the configuration.")
```

Reply:
Same as above, this is not optional in the node pools. If node pools are used, this has to be set there and not here.

Reply:
(I applied the rest of the change)